netket/jax/utils.py

```diff
@@ -79,7 +79,7 @@ def is_complex(x):
     """
     Returns True if x has a complex dtype
     """
-    return jnp.issubdtype(x.dtype, jnp.complexfloating)
+    return jax.numpy.iscomplexobj(x)
```

Comment:
Shouldn't we just drop this function then?

Reply:
fair point.

Reply:
done